runner/server/routes.go

```diff
@@ -1,9 +1,14 @@
 package server
 
+import "net/http"
+
 func (s *Server) routes() {
-	s.router.Methods("POST").Path("/v1/runner/project").HandlerFunc(s.handleCreateProject())
-	s.router.Methods("POST").Path("/v1/runner/service").HandlerFunc(s.handleServiceRequest())
-	s.router.Methods("POST").Path("/v1/runner/events/deploy-service").HandlerFunc(s.HandleApplyService())
+	s.router.Methods(http.MethodPost).Path("/v1/runner/project").HandlerFunc(s.handleCreateProject())
+	s.router.Methods(http.MethodPost).Path("/v1/runner/service").HandlerFunc(s.handleServiceRequest())
+
+	s.router.Methods(http.MethodPost).Path("/v1/runner/{projectId}/service").HandlerFunc(s.HandleApplyService())
+	s.router.Methods(http.MethodGet).Path("/v1/runner/{projectId}/service/{serviceId}/{version}").HandlerFunc(s.HandleApplyService())
+	s.router.Methods(http.MethodDelete).Path("/v1/runner/{projectId}/service/{serviceId}/{version}").HandlerFunc(s.HandleApplyService())
```

Comment:
I think something is wrong over here. We had two handlers to apply services
- First one was used when the client made a request directly (`HandleServiceRequest`)
- Second one was used when the artifact store triggered a webhook (`HandleApplyService`)